### DISCUSSION TITLE
Seem unable to get connect devices

### DISCUSSION THREAD

**Discussion by ovizii** [thlucas1/spotifyplus_card#10] [General]
I installed the card, added it do my dashboard and configured it. I can't select a device to connect to. Can't copy the error message content hence the screenshot.

![image](https://github.com/user-attachments/assets/72cf53fd-2f54-48ca-8154-b1c7d0b41ea1)

These settings I haven't touched.

![image](https://github.com/user-attachments/assets/cad51733-9e30-434b-8d05-01c2592e0e10)

Going to “recently played” I can instantly start playback with this Spotify Plus Card. Not sure  whats up with the above error message.

P.S. Remember, Spotify Plus and Spotify Connect add-on both run on the same HA instance.

**Comment by thlucas1** [comment]
@ovizii 
Can you try clicking on the refresh icon?  It's the little circle icon with an arrow , located to the right of the `filter by device name` entry field.
It appears that the integration cannot find any Spotify Connect devices in the cache.

You might also ensure that you have the latest v1.0.61 release of then SpotifyPlus integration installed.  It contains an updated method to pull device list details that the SpotifyPlus Card UI is using.

Also, can you verify in the media player UI that you normally use (e,g, mini-media player, HA media player, etc) if it lists anything in the `sources` drop down list?

**Reply by ovizii** [reply]
I just checked, and I had 1.0.60 : I updated now. Busy restarting. Will retry. The refresh icon didn't help. Before your card, I used the mini-media player and as you can see, it seems devices.

![image](https://github.com/user-attachments/assets/64ac9a75-b4d3-4d97-a871-fda0cea5b370)

**Reply by ovizii** [reply]
It seems the upgrade to 1.0.61 helped, I now see my device here:

![image](https://github.com/user-attachments/assets/87bcab2d-e5ca-4ae7-9fa7-a3b8ab0cb4f6)

**Comment by ovizii** [comment]
Btw. after the reboot there was a play button here where I marked with yellow. I clicked it and got an error something like failed to perform action or something along the lines (it faded away before I could take a screenshot.

![image](https://github.com/user-attachments/assets/2c887a4d-3839-43b8-83dd-de89b9e77f53)

**Comment by steveguitar78** [comment]
Hello. i see my spotify connect device (Marantz) and Amazon fire stick. i don't see all Google nest devices (Google cast). Why? Can you help me please?

**Reply by thlucas1** [reply]
Chromecast devices are currently not fully supported by the SpotifyPlus integration.  You can perform player functions (e.g. pause, resume, next track, etc) if the device is the currently active player, but you can’t wake up a device or initiate play of tracks / context.

**Reply by steveguitar78** [reply]
thank you
